### ISSUE TITLE
is it depreciated?

### ISSUE THREAD

**Issue by mohd7469** [actions-on-google/dialogflow-interactive-canvas-nodejs#9]
is this node library depreciated as because of the new action on google interactive canvas builder?

**Comment by taycaldwell** [comment]
Although we recommend using Actions Builder to build your Action, the Dialogflow client library is still being maintained if you wish to use Dialogflow.

**Comment by mohd7469** [comment]
But it doesn't support canvas response because in action project there is
no longer option visible to checked that out project uses interactive
response

**Comment by taycaldwell** [comment]
Hi @awais786327,

From the Actions Console, if you navigate to Deploy > Additional Information, you should see a checkbox to enable your Action to use Interactive Canvas. If you do not see this checkbox, please ensure the category (also under Additional Information) for your Action is 'Games & fun'.

**Comment by mohd7469** [comment]
Yes this is what i am asking for, i don't see any checkbox option visible
under Deploy > Additional Information infact my project is created under
using games & fun category. Would you mind if you can confirm if you see
this option on any of your google action projects your side?

**Comment by taycaldwell** [comment]
I do see the option on my side, but I will report this bug internally for investigation.

**Comment by mohd7469** [comment]
In newly created project or in an old one?

**Comment by myhrmans** [comment]
I face the same issue. Instructions do not match on any step for me. I don't see the same menus as described.
